stories/Footer.story.jsx

```diff
@@ -131,42 +131,4 @@ storiesOf("Footer", module)
         </Container>
       </Footer>
     ))
-  )
```

Comment:
all of the stories have no border, to add border you need to explicitly add a css class